schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/history/SubscriptionHistory.java

```diff
@@ -0,0 +1,28 @@
+package com.jasify.schedule.appengine.model.history;
+
+import com.google.appengine.api.datastore.Key;
+import org.slim3.datastore.Model;
+
+/**
+ * @author wszarmach
+ * @since 26/08/15.
+ */
+@Model
+public class SubscriptionHistory extends History {
+    private Key subscriptionId;
```

Comment:
Are you sure, getOrNull calls AsyncDatastoreDelegate.getAsync(Transaction tx, Key id) which has a check to see if key == null and throws NullPointerException if it is